examples/compute-management/02-VirtualMachineCreation.sh

```diff
@@ -3,15 +3,15 @@ set -e
 printf "\n=== Managing Virtual Machine Creation in Azure Compute ===\n"
 
 printf "\n1. Creating a new resource group: %s and location: %s.\n" "$groupName" "$location"
-az group create -n "$groupName" --location "$location"
+az resourcemanager group create -n "$groupName" --location "$location"
```

Comment:
@BurtBiel  The decision we came to this morning was to use azure resourceGroup [verb]